bedrock/firefox/templates/firefox/os/ver/2.5.html

```diff
@@ -15,7 +15,7 @@
     <h1>{{ _('Experience Firefox OS on your Android device')}}</h1>
     <p>{{_('Introducing Firefox OS 2.5 Developer Preview, an experimental app that lets you use Firefox OS on your Android device.')}}</p>
     <div class="primary-cta">
-      <a href="https://d2yw7jilxa8093.cloudfront.net/B2GDroid-mozilla-central-nightly-latest.apk" class="cta-button" data-id="Mozilla_B2G_Activity_Tag_1">{{_('Get the Android App')}}</a>
+      <a href="https://d2yw7jilxa8093.cloudfront.net/B2GDroid-mozilla-central-nightly-latest.apk" class="cta-button" id="52161" data-id="Mozilla_B2G_Activity_Tag_1">{{_('Get the Android App')}}</a>
```

Comment:
Nit: HTML5 is a bit more relaxed I think, but traditionally id's can't start with a number. Even if this does work OK, I would perhaps still favor using a data attribute for this sort of use case. Something like `data-pixel` perhaps?